server.py

```diff
@@ -31,14 +31,29 @@ def api_book_postings(b):
     return json.dumps([])
 
 
+@app.route('/api/book/<b>/postings/new')
```

Comment:
Only GET is allowed by default so the decorator will need @app.route('/api/user/<u>/messages/new', methods=['GET', 'POST'])